api/src/main/java/org/openmrs/api/UserService.java

```diff
@@ -578,4 +578,11 @@ public List<User> getUsers(String name, List<Role> roles, boolean includeRetired
 	 * @since 2.3.6, 2.4.6, 2.5.4, 2.6.0
 	 */
 	Locale getDefaultLocaleForUser(User user);
+
+	/**
+	 * @param user the user to retrieve last-login-time
+	 * @return timestamp representing last login time
```

Comment:
As part of the return description, can we also include an example value to show the timestamp format?